### PR TITLE
initial burrow gtk

### PR DESCRIPTION
**Pull request by reesericci**

### CODE REVIEW

File: burrow-gtk/README.md
Comment (conradev) [inline burrow-gtk/README.md:1]: Let's delete this file for now

File: burrow-gtk/com.hackclub.Burrow.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "burrow-gtk",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "file:///var/home/reesericci/Code"
```
Comment (conradev) [inline burrow-gtk/com.hackclub.Burrow.json:46]: Looks like we might want to make this relative somehow

File: burrow-gtk/data/com.hackclub.Burrow.appdata.xml.in [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.hackclub.Burrow.desktop</id>
```
Comment (conradev) [inline burrow-gtk/data/com.hackclub.Burrow.appdata.xml.in:3]: I've been using lowercase `b` so far, `com.hackclub.burrow`

**Review by conradev — APPROVED** [review]

**Comment by reesericci** [comment]
Build succeeded & all comments resolved. Ready for review

**Comment by reesericci** [comment]
Closes #83

**Review by conradev — COMMENTED** [review]
How do I build and test the app? Would it be possible to add that info to `GETTING_STARTED.md`?

Can we delete `burrow-gtk/README.md`?

File: burrow-gtk/src/config.rs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pub static VERSION: &str = "0.1.0";
+pub static GETTEXT_PACKAGE: &str = "burrow-gtk";
+pub static LOCALEDIR: &str = "/app/share/locale";
+pub static PKGDATADIR: &str = "/app/share/burrow-gtk";
```
Comment (Muirrum) [inline burrow-gtk/src/config.rs:4]: curious where these magic values come from?

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by reesericci** [comment]
I'm not really working on this anymore - someone can feel free to fork & send in a new PR if they want to work on it.

**Pull request closed without merging**